### PR TITLE
Blocks and blocks

### PR DESCRIPTION
**Pull request by vendveri**
Did a bunch of stuff and not sure my commits are the greatest

TL;DR

-  I added logically named blocks in Contentful and reused them when possible
- I gave you examples of gatsby images for fluid and fixed using fragments
- I found a way to export the GraphQL into it's own file and I used folders for pages in Gatsby to group the css externally from the page - it was getting annoying scrolling so much so I had to break the page up to make me not go insane - lol
- I used the same image in contentful for all sections because I know it's just a placeholder and will be replaced
- Not sure how the process page will go with icons so we can chat about that sometime
- I try to comment and name destructured variables with a consistently named alias
- I structured the sections in the order they appear on the page
- I started the runbook for Bryan and will share it with you as I wrap it up
- I used a gatsby plugin to translate contentful markdown into HTML

### CODE REVIEW

**Review by vendveri — COMMENTED** [review]
This worked fine in develop mode but broke on build - it had to do with styled-components inside a gatsby page folder

I used a combination of these two articles: 

[Use custom directory structure in `/pages`](https://github.com/gatsbyjs/gatsby/issues/3800)

[ERROR #11328 A page component must export a React component ... undefined](https://github.com/gatsbyjs/gatsby/issues/24860)

to create a folder in pages and export graphql and styled components for better structure and maintenance

So this code will work in gatsby develop but there are still problems in the build due to GraphQL not being defined inside the home folder - so I'll troubleshoot that issue but you can be free to work with the code in develop mode

**Comment by vendveri** [comment]
https://github.com/gatsbyjs/gatsby/issues/27607

Remove google prefetch

It [appears this plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-prefetch-google-fonts/) is deprecated (doesn't say on gatsby plugin site but people are saying it is deprecated online)

solution
* install gatsby-plugin-preload-fonts
* why do we need to prefetch fonts (there is a split second time when the site loads and the fonts come up - so it creates a design hinderence for a split second (i think this is what caused the glitch in the fonts large and then small when page loads))

Use this plugin - https://www.gatsbyjs.com/plugins/gatsby-plugin-preload-fonts/ for gatsby and this is better as it works with google fonts, adobe fonts...

1. add to packagon.json

```
{
  "scripts": {
    "preload-fonts": "gatsby-preload-fonts"
  }
}
```

remove google fetch plugin and add gatsby-plugin-preload-fonts to gatsby-config.js

Then you need to start up gatsby dev server and then run `$ npm run preload-fonts` in a separte window

if you have no routes you don't need to do this
it will go through typekit/google fonts... and scrape which fonts you are using and write into the cache what to preload

* Added this to Global Styles styled-components

```
*
=============== 
Google Fonts
===============
*/
@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');

/*
=============== 
Global Styles
===============
*/
```
* Now when you run gatsby develop and run `$ npm run preload-fonts` in a separte window you'll see our two fonts added to the cache
